sql_export_email/ir_actions_view.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record id="view_server_action_form" model="ir.ui.view">
+            <field name="name">Server Action (sql_export_email)</field>
+            <field name="model">ir.actions.server</field>
+            <field name="inherit_id" ref="base_export_email.view_server_action_form"/>
+            <field name="arch" type="xml">
+                <field name="state" position="attributes">
+                    <attribute name="on_change">onchange_state(state, context)</attribute>
+                </field>
+                <field name="model_id" position="attributes">
+                    <attribute name="attrs">{'invisible': [('state', '=', 'sql_export_email')]}</attribute>
+                </field>
+                <field name="srcmodel_id" position="attributes">
+                    <attribute name="attrs">{'required': [('state','!=','sql_export_email'),
+                                                          ('state','!=','export_email'),
+                                                          ('state','!=','dummy'),
+                                                          ('state','!=','sms'),
+                                                          ('state','!=','code'),
+                                                          ('state','!=','loop'),
+                                                          ('state','!=','trigger'),
+                                                          ('state','!=','object_copy'),
+                                                          ('state','!=','client_action'),
+                                                          ('state','!=','email'),
+                                                          ('state','!=','sms'),
+                                                          ('state','!=','other')]}</attribute>
```

Comment:
Yes, but I prefer to keep the same syntax as original view :
https://github.com/odoo/odoo/blob/7.0/openerp/addons/base/ir/ir_actions.xml#L358

I hope to find a better way to write this (need to explore fields_view_get)